reco_utils/common/gpu_utils.py

```diff
@@ -15,14 +19,76 @@ def get_number_gpus():
     except CudaSupportError:
         return 0
 
-    
+
 def clear_memory_all_gpus():
-    """Clear memory of all GPUs.
-    """
+    """Clear memory of all GPUs."""
     try:
         for gpu in cuda.gpus:
             with gpu:
                 cuda.current_context().deallocations.clear()
     except CudaSupportError:
         print("No CUDA available")
-        
+
+
+def get_cuda_version():
+    """Get CUDA version
+    
+    Returns:
+        str: Version of the library.
+    """
+    if sys.platform == "win32":
+        raise NotImplementedError("Implement this!")
+    elif sys.platform == "linux" or sys.platform == "darwin":
+        path = "/usr/local/cuda/version.txt"
+        if os.path.isfile(path):
+            with open(path, "r") as f:
+                data = f.read().replace("\n", "")
+            return data
+        else:
+            return "No CUDA in this machine"
+    else:
+        raise ValueError("Not in Windows, Linux or Mac")
+
+
+def get_cudnn_version():
+    """Get the CuDNN version
+    
+    Returns:
+        str: Version of the library.
+    """
+
+    def find_cudnn_in_headers(candiates):
+        for c in candidates:
+            file = glob.glob(c)
+            if file:
+                break
+        if file:
+            with open(file[0], "r") as f:
+                version = ""
+                for line in f:
+                    if "#define CUDNN_MAJOR" in line:
+                        version = line.split()[-1]
+                    if "#define CUDNN_MINOR" in line:
+                        version += "." + line.split()[-1]
+                    if "#define CUDNN_PATCHLEVEL" in line:
+                        version += "." + line.split()[-1]
+            if version:
+                return version
+            else:
+                return "Cannot find CUDNN version"
+        else:
+            return "No CUDNN in this machine"
+
+    if sys.platform == "win32":
+        candidates = [r"C:\NVIDIA\cuda\include\cudnn.h"]
```

Comment:
I think the default for cuda toolkit (where [cuDNN instructions](https://docs.nvidia.com/deeplearning/sdk/cudnn-install/index.html) tell you to put the header) on windows is usually:
C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\vX.X\include
where X.X is the cuda toolkit version (e.g. 9.2)

Reply:
added in the next commit